### PR TITLE
TPD-2370: Upload every value in Nebula packet version 0106

### PR DESCRIPTION
**Pull request by gtursi**

### CODE REVIEW

File: src/timestream-writer.service.js [hunk 1]
```diff
@@ -19,18 +22,70 @@ const writeClient = new AWS.TimestreamWrite({
...
+    .concat(reactivePowerRecords)
+    .concat(apparentPowerRecords)
+    .concat(fundamentalPowerRecords)
+    .concat(activeEnergyRecords)
+    .concat(reactiveEnergyRecords)
+    .concat(apparentEnergyRecords)
+    .concat(fundamentalEnergyRecords)
+    .concat(powerFactorRecords)
+    .concat(frequencyRecords)
+    .concat(phaseAngleRecords);
```
Comment (JuanchiRios) [inline src/timestream-writer.service.js:88]: Using lodash:
```
_.union( 
  ampsRecords,
  voltsRecords,
  ctAmpRecords,
  phaseRecords,
  activePowerRecords,
  reactivePowerRecords,
  apparentPowerRecords,
  fundamentalPowerRecords,
  activeEnergyRecords,
  reactiveEnergyRecords,
  apparentEnergyRecords,
  fundamentalEnergyRecords,
  powerFactorRecords,
  frequencyRecords,
  phaseAngleRecords
)
```

File: src/timestream-writer.service.js [hunk 2]
```diff
@@ -45,17 +100,21 @@ export async function writeRecords({ ts, amps, volts, watts, macAddress, armFirm
...
   MeasureValue: `${value}`,
-  MeasureValueType: 'DOUBLE',
+  MeasureValueType: MeasurementType,
   Time,
   TimeUnit: 'SECONDS'
 });
+
+const doubleRecord = requestRecord('DOUBLE');
+
+const varcharRecord = requestRecord('VARCHAR');
```
Comment (JuanchiRios) [inline src/timestream-writer.service.js:120]: Could you group these constants with   
```
const doubleRecordWithTime = doubleRecord(`${ts}`);
const stringRecordWithTIme = varcharRecord(`${ts}`);
```
Or perhaps extracting these utility functions into a separate file.

**Review by JuanchiRios — APPROVED** [review]
Great Job

**Review by agscusselae — APPROVED** [review]

File: src/timestream-writer.service.js (hunk 2 above)
Comment (gtursi) [reply src/timestream-writer.service.js:120]: but the ones with `WithTime` prefix are created on-the-fly given the log timestamp, that's why they are local variables.

**Pull request merged**